pynest/nest/tests/test_sp/test_growth_curves.py

```diff
@@ -339,22 +339,28 @@ def simulate(self):
                     self.se_nest[n_i], self.se_python[sei_i], decimal=5)
 
     def plot(self):
-        pylab.ion()
-        for i, sei in enumerate(self.se_integrator):
-            pylab.figure()
-            pylab.subplot(1, 2, 1)
-            pylab.title('Ca')
-            pylab.plot(self.sim_steps, self.ca_nest[0, :])
-            pylab.plot(self.sim_steps, self.ca_python[i])
-            pylab.legend(('nest', sei.__class__.__name__))
-            pylab.subplot(1, 2, 2)
-            pylab.title('Synaptic Element')
-            pylab.plot(self.sim_steps, self.se_nest[0, :])
-            pylab.plot(self.sim_steps, self.se_python[i])
-            pylab.legend(('nest', sei.__class__.__name__))
-            pylab.savefig('sp' + sei.__class__.__name__ + '.png')
-        raster_plot.from_device(self.spike_detector)
-        pylab.savefig('sp_raster_plot.png')
+        try:
+            import pylab
+            from nest import raster_plot
+        except ImportError:
+            pass
```

Comment:
I would replace this with `return` and then have the `else` part as plain code.

Reply:
This would be even nicer.